packages/web-components/@sfx/sayt/test/sayt.test.ts

```diff
@@ -65,6 +96,210 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('hideCorrectSayt()', () => {
+    it('should call hideSayt() when event specifies correct searchbox ID ', () => {
+      const hideSayt = stub(sayt, 'hideSayt');
+      const isCorrectSayt = stub(sayt, 'isCorrectSayt').returns(true);
+      const event = {};
+
+      sayt.hideCorrectSayt(event);
+
+      expect(isCorrectSayt).to.be.calledOnceWith(event);
+      expect(hideSayt).to.be.calledOnce;
+    });
+
+    it('should not call hideSayt() when event specifies wrong searchbox ID', () => {
+      const hideSayt = stub(sayt, 'hideSayt');
+      const isCorrectSayt = stub(sayt, 'isCorrectSayt').returns(false);
+      const event = {};
+
+      sayt.hideCorrectSayt(event);
+
+      expect(isCorrectSayt).to.be.calledOnceWith(event);
+      expect(hideSayt).to.not.be.called;
+    });
+  });
+
+  describe('isCorrectSayt()', () => {
+    it('should return true if event provides the correct searchbox ID', () => {
+      const searchbox = sayt.searchbox = 'some-searchbox-id';
+      const event = { detail: { searchbox } };
+
+      const result = sayt.isCorrectSayt(event);
+
+      expect(result).to.be.true;
+    });
+
+    it('should return true if event does not provide a searchbox ID', () => {
+      sayt.searchbox = 'some-searchbox-id';
+      const event = { detail: {} };
```

Comment:
Unless there's a compelling reason otherwise, I would put declarations and initializers above other assignments. At least, that's what we've been doing in the other tests.

Reply:
It does look nicer.